src/slices/requests.ts

```diff
@@ -1,28 +1,28 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
-import * as api from '../api/requests'
+import { IRequest } from '../api/requests'
 import { AppThunk } from 'app/store'
 import { RootState } from 'app/rootReducer'
 
 interface IRequestState {
   isLoading: boolean
-  requests: api.IRequest[]
+  requests: IRequest[]
   error: string
 }
 
-const initialState: IRequestState = {
+export const initialState: IRequestState = {
   isLoading: false,
   requests: [],
   error: '',
 }
 
-const requestsSlice = createSlice({
+export const requestsSlice = createSlice({
   name: 'requests',
   initialState,
   reducers: {
-    getRequests(state) {
-      state = initialState
+    getRequests() {
+      return initialState
```

Comment:
The test found a bug in the implementation, and thus this code needed to be changed. Yay tests to the rescue.